### ISSUE TITLE
More informative alt text for images

### ISSUE THREAD

**Issue by GenevieveBuckley** [napari/tutorials#84]
I've noticed that the alt text for all of the images included with markdown in this repository appears to say the same thing: "image". This seems less than helpful.

I'd like if we replaced the alt text with something better, probably along the lines of "image: a more informative of the image content here"

**Comment by sofroniewn** [comment]
Yes, thanks for flagging this. We're definitely not meeting accessibility guidelines here, and will want to improve this
